src/main/java/org/example/javaproglovo/service/OrderService.java

```diff
@@ -0,0 +1,27 @@
+package org.example.javaproglovo.service;
+
+import lombok.AllArgsConstructor;
+import org.example.javaproglovo.converter.OrderConverter;
+import org.example.javaproglovo.dto.OrderDto;
+import org.example.javaproglovo.repository.OrderRepository;
+import org.springframework.stereotype.Service;
+
+@AllArgsConstructor
+@Service
+public class OrderService {
+
+    private OrderRepository orderRepository;
+
+    public OrderDto findById(int id) {
+        return orderRepository.findById(id).map(OrderConverter::toDto).orElseThrow();
+    }
+
+    public OrderDto save(OrderDto orderDto) {
+        var order = orderRepository.save(OrderConverter.toEntity(orderDto));
```

Comment:
what if the order with the same id exists?

Reply:
In this case, he will be updated